autoware_launch/config/system/diagnostics/localization.yaml

```diff
@@ -6,11 +6,38 @@ units:
         link: /autoware/localization/state
       - type: and
         list:
-          - { type: link, link: /autoware/localization/topic_rate_check/transform }
-          - { type: link, link: /autoware/localization/topic_rate_check/pose_twist_fusion }
-          - { type: link, link: /autoware/localization/scan_matching_status }
-          - { type: link, link: /autoware/localization/accuracy }
-          - { type: link, link: /autoware/localization/sensor_fusion_status }
+          - type: link
+            link: /autoware/localization/topic_rate_check/transform
+          - type: link
+            link: /autoware/localization/topic_rate_check/pose_twist_fusion
+          - type: link
+            link: /autoware/localization/scan_matching_status
+          - type: link
+            link: /autoware/localization/accuracy
+          - type: link
+            link: /autoware/localization/ekf_localizer_activation
+          - type: link
+            link: /autoware/localization/ekf_localizer_set_initial_pose
+          - type: link
+            link: /autoware/localization/ekf_localizer_pose_no_update_count
+          # - type: link
```

Comment:
@Motsu-san 
In the diagnostic entries, `ekf_localizer_pose_queue_size` and `ekf_localizer_twist_queue_size` are commented out.
However, they are listed for publishing in `dummy_diag_publisher`.
Is there any particular reason for this?